k8s/values.yaml

```diff
@@ -1,19 +1,15 @@
-replicaCount: 1
-
-image:
-  repository: pennlabs/{{ project_name }}
-  tag: latest
-  pullPolicy: IfNotPresent
-
-service:
-  port: 80
-
-ingress:
-  enabled: true
-  domain: example.com
-
-secrets:
-  enabled: false
-  secretEnv: {{ project_name }}
-
-project_name: {{ project_name }}
+deploy_version: 0.1.1
+image_tag: latest
+
+applications:
+  - name: django
+    image: pennlabs/{{ project_name }}
+    secret: {{ project_name }}
+    ingress:
+      - host: example.com
+        paths: ["/"]
+    extraEnv:
+      - name: BACKEND_DOMAIN
```

Comment:
Should we also provide a sample frontend domain env variable? Just because we have optional configuration for django+react products already

Reply:
hows this look?

Reply:
Actually wait, we're going to provide a different `values.yaml` in a combined template, right? If that's the case, we either don't need to add a frontend env variable or should set it to be the same as the backend one.

Reply:
Hmm you're right. I'll just rewind the commit.